wait_for_copr/__init__.py

```diff
@@ -55,29 +55,22 @@ def wait_for_copr(owner, project, max_tries, interval, dependency, release):
     )
 
     for _ in range(max_tries):
-        copr_package_proxy = client.package_proxy
-
-        if not copr_package_proxy:
-            # Might be caused by a network issue.
-            click.echo("Issue with initiating Copr package proxy. Retrying.")
-            time.sleep(interval)
-            continue
-
         try:
-            built_version = copr_package_proxy.get(
-                owner, project, dependency, with_latest_succeeded_build=True
-            ).builds["latest_succeeded"]["source_package"]["version"]
+            successful_builds = client.build_proxy.get_list(
+                owner, project, dependency, status="succeeded"
```

Comment:
What if "failed"?  I don't want to delay this fix because it makes a lot of sense as-is.  But not sure if it makes sense to wait any longer when the NVR exists in database but is failed.

Reply:
Good point. It will eventually fail because of the timeout, but we can fail earlier => I'll remove the filter from the parameter and exit early when the build failed.